assets/scss/griidc-colors.scss

```diff
@@ -103,3 +103,23 @@ $navbar-light-active-color: color('black');
 $navbar-light-hover-color: theme-color('primary');
 
 $dropdown-link-hover-color: theme-color('primary');
+

```

Comment:
Maybe one day we should move the common things at a higher level?

Reply:
Yes maybe we can have a common file for both themes